Common/include/config_structure.hpp

```diff
@@ -50,6 +51,10 @@
 #include "./option_structure.hpp"
 #include "./datatype_structure.hpp"
 
+#ifdef HAVE_CGNS
+  #include "cgnslib.h"
+#endif
+
```

Comment:
Hey Tim, thanks that is needed for the GetnDim now moved in config_structure.cpp